StringUtils.c

```diff
@@ -140,3 +143,37 @@ char* String_readLine(FILE* fd) {
       at = buffer + bufSize - step;
    }
 }
+
+int xAsprintf(char** strp, const char* fmt, ...) {
+   va_list vl;
+   va_start(vl, fmt);
+   int _r = vasprintf(strp, fmt, vl);
```

Comment:
Please remove the leading underscore. (Originally `xAsprintf` was a macro, so it was about avoiding shadowing)